crate/operator/create.py

```diff
@@ -420,6 +420,8 @@ def get_statefulset_crate_command(
         "-Cauth.host_based.config.0.user": "crate",
         "-Cauth.host_based.config.0.address": "_local_",
         "-Cauth.host_based.config.0.method": "trust",
+        "-Cauth.host_based.config.1.user": "system",
+        "-Cauth.host_based.config.1.address": "10.0.0.0/8",
```

Comment:
We should probably make this configurable, since no guarantee that all k8s is using class A subnets? Also people might want to restrict it further, if they use a different subnet mask in their k8s.

Reply:
> We should probably make this configurable, since no guarantee that all k8s is using class A subnets? Also people might want to restrict it further, if they use a different subnet mask in their k8s.

_configurable_ as in CRD, plus a UI?